chapter_5_practic/task5.cpp

```diff
@@ -0,0 +1,36 @@
+// Write a function called hms_to_secs() that takes three int values�for hours, minutes,
+// and seconds�as arguments, and returns the equivalent time in seconds(type long).
+// Create a program that exercises this function by repeatedly obtaining a time value in
+// hours, minutes, and seconds from the user(format 12:59 : 59), calling the function, and
+// displaying the value of seconds it returns.
+
+#include <iostream>
+
+inline int hms_to_secs(int h, int m, int s);
```

Comment:
and returns the equivalent time in seconds(type long).
type mismatch

Reply:
Fixed
https://github.com/Dmitriy-Romanchuk/master/pull/5/commits/4d85c1da74474df6e17164c77b3385e5ed9c0423